test_scripts/test_generate_lesson.py

```
@@ -0,0 +1,36 @@
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)